NEWS

```diff
@@ -7,11 +7,63 @@
 ##
 #
 
-## Next Release
+## 1.3.0 (February 15, 2018)
+
 Supported platforms:
+- Added TCP, CUDA, and ROCM transports
```

Comment:
Move to a new line:
- Added support for GPU platforms: Nvidia CUDA and AMD ROCM software stacks